File: archive/archive.py
## BLUE
m.triggers.add_triggerzone(dcs.mapping.Point(22143, -73423, m.terrain), 67056, name="Zone Alpha")
m.terrain.airports["Larnaca"].set_blue()
m.static_group(m.country(dcs.countries.CombinedJointTaskForcesBlue.name), "Larnaca Warehouse", dcs.statics.Warehouse.Warehouse,  dcs.mapping.Point(-7746,-209595, m.terrain))
fg = m.flight_group_inflight(m.country(dcs.countries.CombinedJointTaskForcesBlue.name), "BLUE F-16 Group", dcs.planes.F_16C_50, m.terrain.airports["Larnaca"].position, 42000, group_size=4)
fg.late_activation = True
fg = m.flight_group_inflight(m.country(dcs.countries.CombinedJointTaskForcesBlue.name), "BLUE F-16 Payload", dcs.planes.F_16C_50, m.terrain.airports["Larnaca"].position, 42000)
fg.late_activation = True


## RED
m.terrain.airports["Bassel Al-Assad"].set_red()
m.static_group(m.country(dcs.countries.CombinedJointTaskForcesRed.name), "Bassel Al-Assad Warehouse", dcs.statics.Warehouse.Warehouse,  dcs.mapping.Point(42585, 6308, m.terrain))
fg = m.flight_group_inflight(m.country(dcs.countries.CombinedJointTaskForcesRed.name), "RED F-16 Group", dcs.planes.F_16C_50, m.terrain.airports["Bassel Al-Assad"].position, 42000, group_size=4)
fg.late_activation = True
fg = m.flight_group_inflight(m.country(dcs.countries.CombinedJointTaskForcesRed.name), "RED F-16 Payload", dcs.planes.F_16C_50, m.terrain.airports["Bassel Al-Assad"].position, 42000)
fg.late_activation = True

File: archive/archive.lua

local function setupBlue()
    local airwing = AIRWING:New("Larnaca Warehouse object", "1st Derp")
    airwing:Start()
    local vipers = SQUADRON:New("BLUE F-16 Group", 8, "Good Guys")
    vipers:SetSkill(AI.Skill.AVERAGE)
    vipers:AddMissionCapability({ AUFTRAG.Type.CAP })
    vipers:SetTakeoffHot()
    airwing:AddSquadron(vipers)
    airwing:NewPayload(GROUP:FindByName("BLUE F-16 Payload"), -1, { AUFTRAG.Type.CAP })

    local zoneAlpha = ZONE:New("Zone Alpha")
    local missionCAP = AUFTRAG:NewCAP(zoneAlpha)
    missionCAP:SetRepeat(99)
    airwing:AddMission(missionCAP)
end

local function setupRed()
    local airwing = AIRWING:New("Bassel Al-Assad Warehouse object", "1st Derp")
    airwing:Start()
    local vipers = SQUADRON:New("RED F-16 Group", 8, "Bad Guys")
    vipers:SetSkill(AI.Skill.EXCELLENT)
    vipers:AddMissionCapability({ AUFTRAG.Type.CAP })
    vipers:SetTakeoffHot()
    airwing:AddSquadron(vipers)
    airwing:NewPayload(GROUP:FindByName("RED F-16 Payload"), -1, { AUFTRAG.Type.CAP })

    local zoneAlpha = ZONE:New("Zone Alpha")
    local missionCAP = AUFTRAG:NewCAP(zoneAlpha)
    missionCAP:SetRepeat(99)
    airwing:AddMission(missionCAP)
end

setupBlue()
setupRed()